iidm/iidm-converter-api/src/main/java/com/powsybl/iidm/export/ExportOptions.java

```diff
@@ -42,6 +42,8 @@ public enum IidmVersionIncompatibilityBehavior {
 
     private boolean throwExceptionIfExtensionNotFound = false;
 
+    private boolean throwExceptionIfExtensionVersionNotFound = false;
```

Comment:
I'm not sure it is necessary to add another parameter, you can only use `throwExceptionIfExtensionNotFound` (same remark in `XMLExporter`)

Reply:
I thought it is a different case and @sylvlecl agreed. But I can change it.

Reply:
I think we can consider a unique case as I don't really see a use-case where we need to throw an exception for one and ignore the other (and vice versa).